### ISSUE TITLE
1.5.4- Get Started Prompt after Creating a Circle

### ISSUE THREAD

**Issue by reeserj** [coordinape/coordinape#1008]
Part of https://github.com/coordinape/coordinape/milestone/16 

- Build the Get Started Prompt after Creating a circle
[Figma](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=4737%3A12543)

- Update the Overview page to display the "Getting Started" Prompt
  - [Figma](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=6434%3A27984) (note the buttons are different)
   - Getting Started guide link - https://docs.coordinape.com/get-started/get-started

**Comment by CryptoGraffe** [comment]
I'm tackling this as part of the link joining / add members work. It's already impld.

**Comment by CryptoGraffe** [comment]
The second part of this, the Overview Page isn't done yet, but i think we are waiting on a new version of the illustration before doing that one?

**Comment by CryptoGraffe** [comment]
The empty overview page in 1.5 is totally different than the one in the app now, so i think that warrants its own ticket, rather than it being folded into this one about the get started prompt. 

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/95149165/184515218-9293842b-fe9b-41e1-b6fd-94c43150857b.png">
